runtime/parachains/src/paras/mod.rs

```diff
@@ -291,8 +291,16 @@ pub struct ParaGenesisArgs {
 	pub genesis_head: HeadData,
 	/// The initial validation code to use.
 	pub validation_code: ValidationCode,
-	/// True if parachain, false if parathread.
-	pub parachain: bool,
+	/// Parachain or Parathread.
+	pub paratype: ParaType,
+}
+
+/// Distinguishes between Parachain and Parathread
+#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum ParaType {
+	Parachain,
+	Parathread,
```

Comment:
```suggestion
	Parathread,
	Parachain,
```

this will make it SCALE codec compatible with old storage to avoid migration